src/PrestaShopBundle/Resources/views/Admin/Layout/login_layout.html.twig

```diff
@@ -34,21 +34,34 @@
   {% block javascrips %}{% endblock %}
   {% block stylesheets %}{% endblock %}
 </head>
-<body class="lang-{{ ps.isoUser }}{% if ps.isRtlLanguage %} lang-rtl{% endif %} ps_back-office bootstrap">
+
+{# Construct CSS classes based on version numbers #}
+{% set major_version = constant('PrestaShop\\PrestaShop\\Core\\Version::MAJOR_VERSION') %}
```

Comment:
Again.

Reply:
In this case it is more a problem, as I commented above for security reasons we try not to emphasize on the version of the shop It doesn't mean there are no other ways to determine the PrestaShop version of course, but at least it avoids making things too easy/direct

Reply:
it's ok after login to back office, it should be hidden on the login page